client/connection_pool.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/m3db/m3db/generated/thrift/rpc"
 	nchannel "github.com/m3db/m3db/network/server/tchannelthrift/node/channel"
 	"github.com/m3db/m3db/topology"
-	"github.com/m3db/m3x/close"
+	xclose "github.com/m3db/m3x/close"
```

Comment:
Should we get rid of the name override?

Reply:
I prefer keeping it to indicate we're using a `m3x` package